core/include/kernel_defines.h

```diff
@@ -42,21 +42,21 @@
  */
 #if __STDC_VERSION__ >= 201112L
 #   define container_of(PTR, TYPE, MEMBER) \
-        (_Generic((PTR), \
-            const __typeof__ (((TYPE *) 0)->MEMBER) *: \
-                ((TYPE *) ((char *) (PTR) - offsetof(TYPE, MEMBER))), \
-            __typeof__ (((TYPE *) 0)->MEMBER) *: \
-                ((TYPE *) ((char *) (PTR) - offsetof(TYPE, MEMBER))) \
-        ))
+    (_Generic((PTR), \
+              const __typeof__ (((TYPE *)0)->MEMBER) * : \
+              ((TYPE *)((char *)(PTR)-offsetof(TYPE, MEMBER))), \
+              __typeof__ (((TYPE *)0)->MEMBER) * : \
+              ((TYPE *)((char *)(PTR)-offsetof(TYPE, MEMBER))) \
+              ))
```

Comment:
This is somewhat hard to read, but I'm pretty sure the formatter broke some formatting here.